corehq/apps/reports/static/reports/js/case_list.js

```diff
@@ -1,9 +1,21 @@
-hqDefine("reports/js/case_list", ['jquery', 'analytix/js/kissmetrix'], function ($, kissAnalytics) {
-    $(function () {
-        if (document.location.href.match(/reports\/case_list/)) {
-            $(document).on('click', 'td.case-name-link', function () {
-                kissAnalytics.track.event("Clicked Case Name in Case List Report");
-            });
-        }
+hqDefine("reports/js/case_list", ['jquery', 'analytix/js/kissmetrix', 'hqwebapp/js/initial_page_data'],
+    function ($, kissAnalytics, initialPageData) {
+        $(function () {
+            if (document.location.href.match(/reports\/case_list/)) {
+                $(document).on('click', 'td.case-name-link', function () {
+                    kissAnalytics.track.event("Clicked Case Name in Case List Report");
+                });
+                var userTypes = initialPageData.get('user_types');
+                $(document).on('click', '#apply-filters', function () {
+                    kissAnalytics.track.event("Clicked Apply",
+                        {"filters": _.map($("#paramSelectorForm input[name='case_list_filter']").serializeArray()[0].value.split(','),
```

Comment:
Since you're only looking at one element now, it'd be a bit more concise and clear to access the value with `$("...").val()` rather than using `serializeArray`.

Reply:
Makes sense: https://github.com/dimagi/commcare-hq/pull/22079/commits/cba91b7f937487affefdbe683bf03a7bba5a43b9